packages/deployment/lib/kes.js

```diff
@@ -179,9 +196,260 @@ class UpdatedKes extends Kes {
       dest
     ).then(() => {
       this.Lambda.messageAdapterZipFileHash = new this.Lambda(this.config).getHash(src);
-    }).then(() => super.compileCF());
+      return this.superCompileCF();
+    });
+  }
+
+  /**
+   * Modified version of Kes superclass compileCF method
+   *
+   * Compiles a CloudFormation template in Yaml format.
+   *
+   * Reads the configuration yaml from `.kes/config.yml`.
+   *
+   * Writes the template to `.kes/cloudformation.yml`.
+   *
+   * Uses `.kes/cloudformation.template.yml` as the base template
+   * for generating the final CF template.
+   *
+   * @returns {Promise} returns the promise of an AWS response object
+   */
+  async superCompileCF() {
+    const lambda = new this.Lambda(this.config);
+    if (this.config.lambdaProcess) {
+      this.config = await lambda.process();
+    }
+    else {
+      lambda.buildAllLambdaConfiguration();
+    }
+
+    let cf;
+
+    // Inject Lambda Alias values into configuration,
+    // then update configured workflow lambda references
+    // to reference the generated alias values
+    if (this.config.useWorkflowLambdaVersions === true) {
+      if (this.config.oldLambdaInjection === true) {
+        await this.injectOldWorkflowLambdaAliases();
+      }
+      if (this.config.injectWorkflowLambdaAliases === true) {
+        this.injectWorkflowLambdaAliases();
+      }
+    }
+
+    // if there is a template parse CF there first
+    if (this.config.template) {
+      const mainCF = this.parseCF(this.config.template.cfFile);
+
+      // check if there is a CF over
```

Comment:
I feel like this comment and the one above are incomplete (or maybe I am unable to follow) - could you take a look?

Reply:
Oh I see this is copied from https://github.com/developmentseed/kes/blob/master/src/kes.js

Reply:
And it sounds like you have tickets to reduce code duplication so you can ignore this comment

Reply:
Yes, there's a ticket to reduce/refactor this override  into core kes where possible.   The team agreed the duplicate override logic was acceptable tech debt until it's resolved.    CUMULUS-870